tasks/aiida-daemon.yml

```diff
@@ -28,4 +36,14 @@
     masked: false
     daemon-reload: true
     state: restarted
-  when: not (cloud_platform is defined and cloud_platform == 'docker')
+  when: ansible_service_mgr == "systemd"
+
+- name: Start AiiDA Daemon
+  become: true
+  become_user: "{{ root_user }}"
+  command: "{{ daemon_aiida_venv }}/bin/verdi -p {{ aiida_profile_name }} daemon start"
+  vars:
+    daemon_user: "{{ current_user }}"
+    daemon_user_home: "{{ current_user_home }}"
+    daemon_aiida_venv: "{{ aiida_venv |  regex_replace('\\$\\{HOME}', current_user_home) }}"
```

Comment:
how are these variables used here?

Reply:
Cleaned. In fact only the last one is needed.